doc/cephfs/quota.rst

```diff
@@ -86,3 +86,10 @@ To remove a quota::
 
   setfattr -n ceph.quota.max_bytes -v 0 /some/dir
   setfattr -n ceph.quota.max_files -v 0 /some/dir
+
+.. note:: If some CephFS extended attributes are set on a CephFS directory
+   (say ``/some/dir``), running ``getfattr /some/dir -d -m -`` doesn't print
+    those CephFS extended attributes since CephFS kernel and FUSE clients hide
+    this information from listxattr(2) system call. Access a specific CephFS
+    extended attribute by running ``getfattr /some/dir -n ceph.<some-xattr>``
+    instead.
```

Comment:
Suggestions:

>If some CephFS extended attributes

Only some of them? Maybe leave out the word "some" so that we don't imply that this only applies to certain xattrs?

>information from listxattr(2) system call
->
information from the `listxattr(2)` system call

You may query a specific CephFS extended attribute ...

Reply:
@anthonyeleven: What do you think of the idea that we merge this as it is, and then you and I immediately open a PR to tidy up the semantics of this note?

Reply:
sure

Reply:
I'll create a new PR right now. Stand by.

Reply:
There seems to be an error, which has caused the docs build check to fail:

> /home/jenkins-build/build/workspace/ceph-pr-docs/doc/cephfs/quota.rst:92: ERROR: Unexpected indentation.

I'll create a new docs PR and list the two of you as co-authors.